src/reducers/ticketReducers.js

```diff
@@ -53,6 +75,10 @@ export const map = (state = defaultTicketList, action) => {
     case CREATE_INCIDENT_SUCCESS:
     case RECEIVE_INCIDENT:
       return addIncidentToState(state, action.incident)
+    case REMOVE_TICKET:
+      return removeTicketFromState(state, action.id)
```

Comment:
`return state.filter(ticketId=> ticketId!== action.id)`

Reply:
Redux-persist doesn't play nice with doing it that way.